### PR TITLE
time 64bit, test update

### PR DESCRIPTION
**Pull request by C-Treff**
64bit settings for time is mandatory on INtime. Changed the project file.

Test for ecc_test_buffers was unreliable, as the structs were not initialized befor usage.

### CODE REVIEW

File: wolfcrypt/test/test.c [hunk 1]
```diff
@@ -15729,6 +15729,9 @@ int ecc_test_buffers(void) {
     int verify = 0;
     word32 x;
 
+	memset(&cliKey, 0, sizeof(ecc_key));
+	memset(&servKey, 0, sizeof(ecc_key));
```
Comment (dgarske) [inline wolfcrypt/test/test.c]: Would you mind changing this to `XMEMSET` for portability. Also please use 4 spaces, not hard tabs. Thanks!

**Comment by C-Treff** [comment]
replaced memset with XMEMSET, removed tabs, cleanup projects

**Comment by kaleb-himes** [comment]
Thank you for those updates requested by @dgarske. I'll request a licensing terms review to see if we can accept the changes without a contributor agreement in place!

I see no issue with these changes and deem them relevant for review of licensing terms.

Thanks @C-Treff 

- Kaleb

**Comment by kaleb-himes** [comment]
Just a note: This PR is in reference to [zendesk ticket #3902](https://wolfssl.zendesk.com/agent/tickets/3902)

**Comment by kaleb-himes** [comment]
Jenkins, we can accept changes from @C-Treff. This PR is ok to test.

- KH

**Comment by kaleb-himes** [comment]
Changes approved. Thanks @C-Treff for this submission!

- KH

**Pull request merged**